cleverhans/evaluation.py

```diff
@@ -3,6 +3,8 @@
 Includes multi-GPU support for fast evaluation.
 """
 
+import warnings
```

Comment:
Why was this import added?

Reply:
Because module `warnings` is used in `batch_eval`, but it has never been imported. See [here](https://github.com/tensorflow/cleverhans/blob/3b4820ab723db3dae549154064d748695e81805d/cleverhans/evaluation.py#L259) for more details.

Reply:
Thanks for catching that! Could you move the import after six.moves so they are in alphabetical order?

Reply:
Sure!